src/Contract/API/Management/OrganizationsInterface.php

```diff
@@ -229,14 +229,16 @@ public function getByName(
      * Get client grants associated to an organization.
      * Required scope: `read:organization_client_grants`.
      *
-     * @param string              $id      Organization (by ID) that the connection is associated with
-     * @param null|RequestOptions $options Optional. Additional request options to use, such as a field filtering or pagination. (Not all endpoints support these.)
+     * @param string              $id       Organization (by ID) that the connection is associated with
+     * @param string              $grantIds Client Grant (by ID) to associate with the organization.
+     * @param null|RequestOptions $options  Optional. Additional request options to use, such as a field filtering or pagination. (Not all endpoints support these.)
      *
      * @throws \Auth0\SDK\Exception\ArgumentException when an invalid `id` or `connectionId` are provided
      * @throws \Auth0\SDK\Exception\NetworkException  when the API request fails due to a network error
      */
     public function getClientGrants(
         string $id,
+        string $grantIds,
```

Comment:
We should move this new parameter after `$options`; otherwise, it's technically a breaking change on the method signature of the public API.

Reply:
@evansims  I have resolved this. Thanks for the heads up !!